src/rust/src/dep-trees/p22-expect.txt

```diff
@@ -169,35 +168,36 @@ soroban-env-host v22.0.0 (src/rust/soroban/p22/soroban-env-host)
 │   │   │   ├── der v0.7.6 (*)
 │   │   │   ├── generic-array v0.14.7 (*)
 │   │   │   ├── subtle v2.5.0
-│   │   │   └── zeroize v1.6.0 (*)
+│   │   │   └── zeroize v1.8.1 (*)
 │   │   ├── subtle v2.5.0
-│   │   └── zeroize v1.6.0 (*)
+│   │   └── zeroize v1.8.1 (*)
 │   ├── rfc6979 v0.4.0
 │   │   ├── hmac v0.12.1
 │   │   │   └── digest v0.10.7 (*)
 │   │   └── subtle v2.5.0
 │   └── signature v2.1.0
 │       ├── digest v0.10.7 (*)
 │       └── rand_core v0.6.4 (*)
-├── ed25519-dalek v2.0.0
-│   ├── curve25519-dalek v4.1.1 (*)
+├── ed25519-dalek v2.1.1
+│   ├── curve25519-dalek v4.1.3 (*)
 │   ├── ed25519 v2.2.2
 │   │   └── signature v2.1.0 (*)
 │   ├── rand_core v0.6.4 (*)
 │   ├── sha2 v0.10.8
 │   │   ├── cfg-if v1.0.0
 │   │   ├── cpufeatures v0.2.8 (*)
 │   │   └── digest v0.10.7 (*)
-│   └── zeroize v1.6.0 (*)
-├── elliptic-curve v0.13.5 (*)
+│   ├── subtle v2.5.0
+│   └── zeroize v1.8.1 (*)
+├── elliptic-curve v0.13.8 (*)
 ├── generic-array v0.14.7 (*)
 ├── getrandom v0.2.11 (*)
 ├── hex-literal v0.4.1
 ├── hmac v0.12.1 (*)
-├── k256 v0.13.1
+├── k256 v0.13.4
```

Comment:
Shouldn't this be v0.13.3 like we have in latest env?

Reply:
the latest env specifies "0.13.3" in its Cargo.toml file but as mentioned that is implicitly a loose (non-locked) caret-type constraint -- equivalent to "^0.13.3" which resolved to 0.13.4 in the committed env lockfile. Which is why it's 0.13.4 here. This file is literally generated from the env lockfile directly now. It's just another copy of it as a check to catch unanticipated changes.

Reply:
See: https://github.com/stellar/rs-soroban-env/blob/main/soroban-env-host/Cargo.toml#L36 and https://github.com/stellar/rs-soroban-env/blob/main/Cargo.lock#L857